src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/ScoreboardElements.kt

```diff
@@ -0,0 +1,470 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.HypixelData
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.data.MaxwellAPI
+import at.hannibal2.skyhanni.data.MayorElection
+import at.hannibal2.skyhanni.data.PartyAPI
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.data.SlayerAPI
+import at.hannibal2.skyhanni.mixins.hooks.replaceString
+import at.hannibal2.skyhanni.utils.LorenzUtils.inDungeons
+import at.hannibal2.skyhanni.utils.LorenzUtils.nextAfter
+import at.hannibal2.skyhanni.utils.RenderUtils.AlignmentEnum
+import at.hannibal2.skyhanni.utils.StringUtils.firstLetterUppercase
+import at.hannibal2.skyhanni.utils.TimeUtils.formatted
+import at.hannibal2.skyhanni.test.command.ErrorManager
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import java.util.function.Supplier
+
+private val config get() = SkyHanniMod.feature.gui.customScoreboard
+
+// Stats / Numbers
+var purse = "0"
+var motes = "0"
+var bank = "0"
+var bits = "0"
+var copper = "0"
+var gems = "0"
+var location = "None"
+var lobbyCode = "None"
+var heat = "0"
+var mithrilPowder = "0"
+var gemstonePowder = "0"
+var extraLines = listOf<String>()
+
+val extraObjectiveLines = listOf("§7(§e", "§f Mages", "§f Barbarians")
+var amountOfExtraLines = 0
+
+enum class ScoreboardElements(
+    private val displayPair: Supplier<List<Pair<String, AlignmentEnum>>>,
+    private val showWhen: () -> Boolean,
+    private val configLine: String
+) {
+    TITLE(
+        {
+            val alignment = if (config.displayConfig.centerTitleAndFooter) {
+                AlignmentEnum.CENTER
+            } else {
+                AlignmentEnum.LEFT
+            }
+
+            when (config.displayConfig.useHypixelTitleAnimation) {
+                true -> listOf(ScoreboardData.objectiveTitle to alignment)
+                false -> listOf(config.displayConfig.customTitle.get().toString().replace("&", "§") to alignment)
+            }
+        },
+        {
+            true
+        },
```

Comment:
For the Scoreboard elements, could you extract these to functions to make it more readable?